EDL - increase timeout and docs (#26069)
demisto/content: Packs/EDL/Integrations/EDL/EDL_description.md

### When Running in On-Demand Mode
Make sure you run the ***!export-indicators-list-update*** command for the first time to initialize the export process.

### Important Notes:

> - When constantly using different queries for the same EDL instance through the *q* inline argument, it is recommended to instead use different instances of the EDL (one for each query), and set each one with a default query for better performance.

Reviewer: ```suggestion
- If constantly using different queries for the same EDL instance through the *q* inline argument, it is recommended to use different instances of the EDL (one for each query), and set each one with a default query for better performance.
```
